### ISSUE TITLE
Accessibility: unlabeled "+" button in bottom toolbar

### ISSUE THREAD

**Issue by stevefaulkner** [brave/browser-ios#150]
The new tab button "+"  in the bottom toolbar does not have a label announced when focused using VoiceOver - this suggests the button does not have an accessible name provided
